tools/toolchains/arm.py

```diff
@@ -581,10 +581,6 @@ def __init__(self, target, *args, **kwargs):
                 self.flags['common'].append("-D__MICROLIB")
             if "--library_type=microlib" not in self.flags['ld']:
                 self.flags['ld'].append("--library_type=microlib")
-            if "-Wl,--library_type=microlib" not in self.flags['c']:
-                self.flags['c'].append("-Wl,--library_type=microlib")
-            if "-Wl,--library_type=microlib" not in self.flags['cxx']:
-                self.flags['cxx'].append("-Wl,--library_type=microlib")
```

Comment:
Can you confirm that the assembler option `--library_type=microlib` is not giving any warning (as per the docs, it should not be necessary to use the assembler option)?

Reply:
There are startup_MK64F12.S and except.S assembly files built without compiler warning.

Reply:
Okay. Thanks for checking